lib/omniauth/strategies/qualtrics.rb

```diff
@@ -1,14 +1,14 @@
-require 'omniauth-oauth2'
+require "omniauth-oauth2"
```

Comment:
I know this goes against rubocop guidelines, but this is the request from repo maintainer

Reply:
when forking, you should stick to the rubocop of the project if there is one rather than ours

Reply:
and just generally make the diff as small as possible

Reply:
> I know this goes against rubocop guidelines, but this is the request from repo maintainer

Yeah it looks like it's consistent with their own [rubocop config](https://github.com/user-interviews/omniauth-qualtrics/blob/master/.rubocop.yml). 

I'm not sure why my tooling didn't just use pick up this config automatically, but thanks for fixing 👍